packages/sak-dokumenter/src/components/DocumentList.tsx

```diff
@@ -152,69 +126,102 @@ const DocumentList: FunctionComponent<OwnProps> = ({ documents, behandlingUuid,
-                title={intl.formatMessage({ id: 'DocumentList.Motta' })}
-                height={25}
-                width={25}
-                color="var(--a-surface-success)"
-              />
-            )}
-            {document.kommunikasjonsretning === Kommunikasjonsretning.UT && (
-              <ChevronLeftDoubleIcon
-                title={intl.formatMessage({ id: 'DocumentList.Send' })}
-                height={25}
-                width={25}
-                color="var(--a-orange-500)"
-              />
-            )}
-            {document.kommunikasjonsretning !== Kommunikasjonsretning.INN &&
-              document.kommunikasjonsretning !== Kommunikasjonsretning.UT && (
-                <ChevronDownDoubleIcon
-                  title={intl.formatMessage({ id: 'DocumentList.Intern' })}
-                  height={25}
-                  width={25}
-                  color="var(--a-blue-400)"
-                />
-              )}
-          </TableColumn>
-          <TableColumn>
-            {document.tittel}
-            {document.behandlingUuidList && behandlingUuid && document.behandlingUuidList.includes(behandlingUuid) && (
-              <StarFillIcon className={styles.image} title={intl.formatMessage({ id: 'DocumentList.IBruk' })} />
-            )}
-          </TableColumn>
-          <TableColumn>
-            {document.gjelderFor && isTextMoreThan25char(document.gjelderFor) && (
-              <Tooltip content={<BodyShort size="small">{document.gjelderFor}</BodyShort>} alignLeft>
-                {trimText(document.gjelderFor)}
-              </Tooltip>
-            )}
-            {document.gjelderFor && !isTextMoreThan25char(document.gjelderFor) && document.gjelderFor}
-          </TableColumn>
-          <TableColumn>
-            {document.tidspunkt ? (
-              <DateTimeLabel dateTimeString={document.tidspunkt} />
-            ) : (
-              <BodyShort size="small">
-                <FormattedMessage id="DocumentList.IProduksjon" />
-              </BodyShort>
-            )}
-          </TableColumn>
-        </TableRow>
-      ))}
-    </Table>
+          <FormattedMessage id="DocumentList.LastNedKnapp" values={{ antall: valgteDokumentIder.length }} />
+        </Button>
+      )}
+      <Table size="small" sort={sort} onSortChange={sortKey => handleSort(sortKey as TableHeaders)}>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>
+              <Checkbox
+                checked={valgteDokumentIder.length === sortedDocuments.length}
+                indeterminate={valgteDokumentIder.length > 0 && valgteDokumentIder.length !== sortedDocuments.length}
+                onChange={() =>
+                  valgteDokumentIder.length > 0
+                    ? setValgteDokumentIder([])
+                    : setValgteDokumentIder(sortedDocuments.map(({ dokumentId }) => dokumentId))
+                }
+                hideLabel
+              >
+                Velg alle rader
```

Comment:
{' '} her òg kanskje?

Reply:
Jeg er litt usikker på funksjonen på teksten der. Tipper det bare er relevant for skjermlesere? Eksempelet i Aksel har med den teksten der, så da tipper jeg det er good practice 🤞